src/manifest/manifest_schema.ts

```diff
@@ -296,10 +297,7 @@ export type ManifestOAuth2Schema = {
 
 export type ManifestOAuth2ProviderSchema = {
   provider_type: OAuth2ProviderTypeValues;
-  options: {
-    // deno-lint-ignore no-explicit-any
-    [key: string]: any;
-  };
+  options: OAuth2ProviderOptions;
```

Comment:
This change may be breaking 😅 
The `DefineOAuth2Provider` function requires its `options` parameter to be of type `OAuth2ProviderOptions`, but this may break apps that are defining there `ManifestOAuth2ProviderSchema` manually?